cli/cmd/compliance.go

```diff
@@ -44,6 +45,18 @@ var (
 
 		// display extended details about a compliance report
 		Details bool
+
+		// Filter the recommendations table by category
+		Category []string
+
+		// Filter the recommendations table by service
+		Service []string
+
+		// Filter the recommendations table by severity
+		Severity string
+
+		// Filter the recommendations table by status
+		Status string
```

Comment:
I am assuming that you didn't apply the suggested modifications to the flag `--status`, just like `--category` and `--service`. Any reason why not?